jax/_src/path.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2022 The JAX Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import pathlib
+
+logger = logging.getLogger(__name__)
```

Comment:
What's this? Just use absl logging?

Reply:
Ohh wow.. I don't like the current thing. Why do you have to get the logger instead of doing just logging. We can discuss that separately though.